test/test_build.py

```diff
@@ -338,6 +339,44 @@ def test_should_raise_an_error_when_docker_repository_not_in_lower_case(self):
             """)
             with self.assertRaises(ValueError):
                 bigflow.build.get_docker_repository_from_deployment_config(Path(f.name))
-    
+
     def get_version_error(self):
         raise RuntimeError('get_version error')
+
+
+class PipToolsTestCase(TestCase):
```

Comment:
Please, stay consistent and use given/when/then schema

Reply:
Done. 
BTW, Is this some known/documented practice?

Reply:
It's an approach from [Spock](http://spockframework.org/spock/docs/1.3/spock_primer.html#_given_blocks).